extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -211,8 +220,8 @@ export class IframeTransport {
    * @returns {!string} Unique ID of this instance of IframeTransport
    * @VisibleForTesting
    */
-  getId() {
-    return this.id_;
+  getCreativeId() {
+    return this.creativeId_;
```

Comment:
nit: why not just use `this.creativeId_`?

Reply:
The main reason for this method to exist is so that the unit tests don't have to access IframeTransport.creativeId_.
But since I had to add the method, I went ahead and called it from with IframeTransport also. But I can back that out.

Reply:
Fixed